### PR TITLE
chore(http-polling): introduce template

### PR DESCRIPTION
**Pull request by Oleksiivanov**
## Description

Added HTTP polling template
Screenshot 2023-09-14 at 12 03 40

Screenshot 2023-09-18 at 09 03 52

Screenshot 2023-09-14 at 12 02 28

## Related issues
https://github.com/camunda/team-connectors/issues/449

### CODE REVIEW

File: connectors/http/polling/element-templates/http-polling-connector.json [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+      "feel": "required",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:property",
+        "name": "activationCondition"
+      },
+      "description": "Condition under which the connector triggers. Leave empty to catch all events. <a href='https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/http-webhook/#make-your-http-webhook-connector-for-receiving-messages-executable' target='_blank'>See documentation</a>"
+    },
+    {
+      "label": "Timer",
```
Comment (chillleader) [inline connectors/http/polling/element-templates/http-polling-connector.json:407]: nit: It wasn't immediately clear to me why we need Timer and HTTP request delay at the same time. I understand it now, but maybe we can call this property "Time to live", or "Expiration time"...

Comment (chillleader) [reply connectors/http/polling/element-templates/http-polling-connector.json:407]: Perhaps also a note saying that connector will run indefinitely if you leave this blank

File: connectors/http/polling/element-templates/http-polling-connector.json [hunk 2]
```diff
@@ -0,0 +1,491 @@
...
+        "name": "inbound.timer.message"
+      },
+      "optional": true,
+      "feel": "optional",
+      "constraints": {
+        "notEmpty": false
+      }
+    },
+    {
+      "label": "Connection timeout",
```
Comment (chillleader) [inline connectors/http/polling/element-templates/http-polling-connector.json:437]: I would prefer to move this field into the `endpoint` group (to make it visually close to the HTTP request so users can easily understand the purpose of this field)

**Review by chillleader — COMMENTED** [review]
Looks good 🙌 
Just a few suggestions for properties layout

**Review by igpetrov — COMMENTED** [review]
Overall, looks good to me. I think, you should present the CX to @bastiankoerber sooner than later to make sure we're moving in the right direction.

Just a side note, do we need to add "new features" of the HTTP connector introduced by `0.22.0`, like API keys parsing, response body expression, etc?

**Review by bastiankoerber — APPROVED** [review]
Looks good for me, just was wondering about the use case of http delete/put/patch method during polling.

but for a super generic polling connector it's fine.

File: connectors/http/polling/element-templates/http-polling-connector.json [hunk 3]
```diff
@@ -0,0 +1,460 @@
...
+      "type": "Text",
+      "feel": "required",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "headers"
+      },
+      "optional": true
+    },
+    {
+      "label": "HTTP request delay",
```
Comment (sbuettner) [inline connectors/http/polling/element-templates/http-polling-connector.json:181]: Lets call it `interval` as its state in the description of the connector.

**Pull request merged**